Directory.Build.props

```
@@ -20,6 +21,10 @@
     <Deterministic>true</Deterministic>
     <!--This will target the latest patch release of the runtime released with the current SDK.  -->
     <TargetLatestRuntimePatch>true</TargetLatestRuntimePatch>
+  </PropertyGroup>
+  <!-- Shared dependencies versions.-->
+  <PropertyGroup>
+    <IdentityModel>6.10.2</IdentityModel>
```

nit: Keep version suffix for consistency?
```suggestion
    <IdentityModelVersion>6.10.2</IdentityModelVersion>
```